docs/user-management/create-manage-users.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebar_position: 2
+title: Create and Manage Users
+---
+
+This section describes how to create and edit user accounts.
+
+## Create User
+
+Follow these instructions to create a new user using the GDN console web UI.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Account**.
+1. On the **Users** tab, click **New User**.
+1. Enter a unique username.
+1. Enter the email to be associated with this user account.
+1. Enter a password and confirm it.
+1. Optionally, choose to create the account as inactive by deselecting **Active**.
+
+## Change Password
+
+On the **Users** tab, you can select a user to access their details and change their password.
+
+1. [Log in to your Macrometa account](https://auth.paas.macrometa.io/).
+1. Click **Account**.
+1. On the **Users** tab, select the user whose password you want to change.
+1. Click **Edit** in the **Password** field.
+1. Enter and confirm the new password, and click **Save**.
```

Comment:
```suggestion
1. Enter and confirm the new password, then click **Save**.
```

Reply:
Careful with "and." It can suggest that something happens at the same time and confuse non-native English readers. In this case, you do things in sequence, so signal that with "then."

Apply throughout.